de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/auth/filter/DynamicRepositoryFilterBinding.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server.auth.filter;
+
+import javax.ws.rs.container.DynamicFeature;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.FeatureContext;
+
+import de.dlr.sc.virsat.server.resources.AuthTestResource;
+
+/**
+ * Binds the RepositoryFilter to specified classes and functions,
+ * so the filter will only be used on those
+ */
+public class DynamicRepositoryFilterBinding implements DynamicFeature {
+
+	@Override
+	public void configure(ResourceInfo resourceInfo, FeatureContext context) {
+		if (AuthTestResource.class.equals(resourceInfo.getResourceClass())

```

Comment:
Okay for a temporary hack, but on the long run I think the check should be something like if query string contains "/repository/" or something like this, so when we add more resources that conform to this scheme, they will have filtering

Reply:
Yes that's what we should do.